asteroid/engine/system.py

```diff
@@ -33,21 +35,27 @@ class System(pl.LightningModule):
     """
 
     def __init__(
-        self,
-        model,
-        optimizer,
-        loss_func,
-        train_loader,
-        val_loader=None,
-        scheduler=None,
-        config=None,
+            self,
+            model,
+            optimizer,
+            loss_func,
+            train_loader,
+            sample_rate,
+            val_loader=None,
+            test_loader=None,
+            metrics_list=[],
+            scheduler=None,
+            config=None,
```

Comment:
Please don't change the indent. It's harder to see what has been changed than if the indent isn't changed.